01-JS-DOM-APIs/exerciseJS.js

```diff
@@ -0,0 +1,75 @@
+//EXERCISE 6
+  function eventRequest() {
+    var xhttp = new XMLHttpRequest();
+    xhttp.open('GET', 'http://api.icndb.com/jokes/random', true);
+    xhttp.onreadystatechange = function(event) {
+      var response = JSON.parse(event.target.response);
+      document.getElementById('sectionHidden').innerHTML = response.value.joke;
+       if (xhttp.readyState === XMLHttpRequest.DONE) {
```

Comment:
It is a good practice to leave a break line between the if block and the line above